packages/components/src/ActionList/ActionList.tsx

```diff
@@ -112,11 +116,42 @@ export const ActionListLayout: FC<ActionListProps> = ({
   header = true,
   children,
   columns,
-  itemsSelected,
-  onClickRowSelect,
+  itemsSelected = [],
+  onClickRowSelect = false,
   onSelect,
+  onSelectAll,
   onSort,
 }) => {
+  const [allItems, setAllItems] = useState([] as string[])
```

Comment:
```suggestion
  const [allItems, setAllItems] = useState<string[]>([])
```

Using `as` is dangerous because Typescript will just believe whatever you tell it:

```js
  const [allItems, setAllItems] = useState([] as WhateverYouSayHereIsTheTypeNow)
```

Reply:
Resolved in 76dfb2d!